scripts/generate-contributors-data.mjs

```
@@ -0,0 +1,129 @@
+import { mkdir, writeFile } from 'node:fs/promises';
+import { dirname } from 'node:path';
+
+const [repo = 'helmforgedev/charts', outputPath = 'src/data/contributors.ts'] = process.argv.slice(2);
```

ESLint

[error] 'process' is not defined.

(no-undef)